cocotb/share/lib/embed/embed.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright cocotb contributors
+// Licensed under the Revised BSD License, see LICENSE for details.
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <embed.h>
+#include <cocotb_utils.h>   // xstr, utils_dyn_open, utils_dyn_sym
+#include <gpi.h>            // gpi_event_t
+#include <cstdlib>          // getenv
+#if ! defined(__linux__) && ! defined(__APPLE__)
+#include <windows.h>        // Win32 API for loading the embed impl library
+#include <string>           // string
+#endif
+
+#ifndef EMBED_IMPL_LIB
+#error "Name of embed implementation library required"
+#else
+#define EMBED_IMPL_LIB_STR xstr(EMBED_IMPL_LIB)
+#endif
+
+
+static void (*_embed_init_python)(void);
+static void (*_embed_sim_cleanup)(void);
+static int (*_embed_sim_init)(int argc, char const * const * argv);
+static void (*_embed_sim_event)(gpi_event_t level, const char *msg);
+
+static bool init_failed = false;
+
+
+#define PRINT_ERROR_WINDOZE(log_fmt) {\
```

Comment:
```suggestion
#define PRINT_ERROR_WINDOWS(log_fmt) {\
```
The original doesn't age well.